.circleci/config.yml

```diff
@@ -1,100 +1,99 @@
 version: 2
 jobs:
-  test:
-    docker:
-      - image: circleci/python:3.6.1
-      - image: circleci/postgres:9.4
-        environment:
-          - TERM: xterm-256color
-          - POSTGRES_USER: circleci
-          - POSTGRES_DB: circlecidb
-          - POSTGRES_PASSWORD: $DB_PASSWORD
-    working_directory: ~/andela-socials-backend
+    test:
+        docker:
+            - image: circleci/python:3.6.1
+            - image: circleci/postgres:9.4
+              environment:
+                  - TERM: xterm-256color
+                  - POSTGRES_USER: circleci
+                  - POSTGRES_DB: circlecidb
+                  - POSTGRES_PASSWORD: $DB_PASSWORD
```

Comment:
The env variable did not reflect in the file and that is why probably it did not show up on CircleCi